### PR TITLE
feat(appeals): add application outcome change page

### PR DESCRIPTION
**Pull request by AlexMoulin**
## Issue ticket number and link
https://pins-ds.atlassian.net/browse/A2-296
## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [x] I have made corresponding changes to the documentation
- [x] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [x] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: appeals/web/src/server/appeals/appeal-details/application-outcome/__tests__/application-outcome.test.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+const request = supertest(app);
+const appealId = appealData.appealId;
+const appellantCaseId = appealData.appellantCaseId;
+const baseUrl = `/appeals-service/appeal-details/${appealId}/appellant-case`;
+
+describe('application-outcome', () => {
+	beforeEach(installMockApi), afterEach(teardown);
+
+	describe('GET /change', () => {
+		it('shouldrender the applicationOutcome change page', async () => {
```
Comment (james-hill-kc) [inline appeals/web/src/server/appeals/appeal-details/application-outcome/__tests__/application-outcome.test.js:17]: Super pedantic but there's a missing space in the test name here: `shouldrender`

File: appeals/web/src/server/appeals/appeal-details/application-outcome/application-outcome.mapper.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+							isPageHeading: false,
+							classes: 'govuk-fieldset__legend--1'
+						}
+					},
+					items: [
+						{
+							value: 'granted',
+							text: 'Granted',
+							checked:
+								storedSessionData === 'granted' ||
```
Comment (james-hill-kc) [inline appeals/web/src/server/appeals/appeal-details/application-outcome/application-outcome.mapper.js:39]: This seems like it allows `applicationDecision` to override the `storedSessionData` value even if the latter is set. Is this intentional?

**Review by james-hill-kc — APPROVED** [review]
Looks good, I've left a couple of comments

**Review by james-hill-kc — COMMENTED** [review]
Mistakenly approved when I meant to comment...

**Review by james-hill-kc — DISMISSED** [review]

**Review by glpins — APPROVED** [review]

**Pull request merged**